### PR TITLE
Support get org info by org name, add new domain to whitelist, small code refactor, fix typo.

### PR DESCRIPTION
**Pull request by shuofan**
#### What type of PR is this?
feature

#### What this PR does / why we need it:
Adapt new domain and url directory. 
#### Which issue(s) this PR fixes:

Fixes #

#### Specified Reivewers:
@Effet 

#### Does this PR introduce a user-facing change?

```release-note

```

#### Additional documentation usage docs, etc.:

```docs

```

### CODE REVIEW

File: modules/cmdb/services/org/org.go [hunk 1]
```diff
@@ -465,10 +465,31 @@ func (o *Org) List() ([]model.Org, error) {
 	return o.db.GetOrgList()
 }
 
+// Get Org by domain and org name
+func (o *Org) GetOrgByDomainAndOrgName(domain, orgName string) (*model.Org, error) {
```
Comment (sfwn) [inline modules/cmdb/services/org/org.go:469]: unit test requested.

**Review by sfwn — CHANGES_REQUESTED** [review]

File: modules/cmdb/conf/conf.go [hunk 2]
```diff
@@ -178,10 +178,11 @@ func Load() {
 	fmt.Println(fileMaxUploadByte.String())
 	cfg.FileMaxUploadSize = fileMaxUploadByte
 
-	OrgWiteList = map[string]bool{
+	OrgWhiteList = map[string]bool{
 		UIDomain():                          true,
 		OpenAPIDomain():                     true,
 		"openapi.default.svc.cluster.local": true,
+		"www.erda.cloud":                    true,
```
Comment (Effet) [inline modules/cmdb/conf/conf.go:185]: do not use hard-coded domain, UIDomain is enough

**Review by Effet — CHANGES_REQUESTED** [review]

File: modules/cmdb/conf/conf.go (hunk 2 above)
Comment (shuofan) [reply modules/cmdb/conf/conf.go]: Done

File: modules/cmdb/services/org/org.go (hunk 1 above)
Comment (shuofan) [reply modules/cmdb/services/org/org.go:471]: done

**Review by Effet — APPROVED** [review]

**Review by sfwn — APPROVED** [review]

**Pull request merged**